design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,47 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task, it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review, but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview / Problem Statement
+
+* Describe the task with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
```

Comment:
Could this be replaced with a link back to the user story? The user story should capture the what is needed and the justification. I would not want to have two sources of truth for that. So maybe remove this bullet and add to the placeholder's one for the user story?

Reply:
I agree, but at the same time i feel it is important for historical purposes.  Think of the scenario where this design review could be leveraged by a future engagement and we no longer have access to the user story in the customers account.

Reply:
I have added the flexible option to link to the task so we can have the details even if the link is unavailable.